### PR TITLE
Add require-oslogin-certificates logic to disable keys

### PR DESCRIPTION
**Pull request by bpl4vv**
Add logic to read `require-oslogin-certificates` from metadata and disable keys if true

### CODE REVIEW

File: google_guest_agent/oslogin.go [hunk 1]
```diff
@@ -312,12 +320,15 @@ func updateSSHConfig(sshConfig string, enable, twofactor, skey bool) string {
 	if enable {
 		osLoginBlock := []string{googleBlockStart}
 
-		if cfg.Get().OSLogin.CertAuthentication {
+		// Metadata overrides the config file.
```
Comment (ChaitanyaKulkarni28) [inline google_guest_agent/oslogin.go:323]: I guess this can be simplified by - 

```
if  reqCerts ||  cfg.Get().OSLogin.CertAuthentication {
 ---
}

**Review by dorileo — APPROVED** [review]

**Pull request merged**